sdk/spring/spring-reference.yml

```diff
@@ -1,847 +0,0 @@
-# Please refer to this link to see design details and how to change:
```

Comment:
How about not change this file, just sync from main branch. Otherwise, we need resolve the conflicts.

Reply:
Currently, we don't need to sync from main to Spring boot 3 branch,  the sync job should be decided by the PR target. I prefer to delete this one, the conflicts will be only one time when Spring Boot 3 branch is merged back to the main branch.